README.md

```diff
@@ -62,17 +59,24 @@ config.skyMap.patchInnerDimensions=[4000, 4000]
 config.skyMap.pixelScale=0.263
 ```
 
-hsc
----
+## hsc
 
 Source catalogs, metadata, and zeroed+compressed images taken from the `w_2020_14`processing run of the HSC RC dataset, available at `lsst-dev:/datasets/hsc/repo/rerun/RC/w_2020_14/DM-24359-sfm`.
 The included `scripts/compress_jointcal_hsc_test_data.py` file copies the necessary data from the butler repo, removes pixel-level data from the calexps and compresses the source catalogs, and extracts the relevant portions of the sqlite3 registry.
 The src catalogs have been converted to sourceTable_visit parquet tables for quick ingest.  The included `scripts/make_sourcetables_jointcal_hsc_test_data.sh` file runs the appropriate transformation tasks and cleans up temporary files.
 The raw image files, which are necessary for gen3 conversion, are copied into the repo by `scripts/compress_jointcal_hsc_raw_test_data.py` which copies in the necessary data, and removes pixel-level data from the raws.
-The hsc gen2 repo can be converted to a gen3 repo with the `scripts/convert_gen2_to_gen3_hsc.sh` script, which does the conversion and slims down the associated sky map.  The gen3 repo can be exported to `hsc/exports.yaml` with `scripts/export_gen3_hsc.py`.  The output gen3 files will not be committed to the repo until the gen3 API is determined to be stable enough.
 
-Git LFS
--------
+### gen3 conversion
+
+To create a gen3 repo from the existing gen2 repo and make the exports file that is used to regenerate the gen3 repo for tests:
+
+* Run `scripts/convert_gen2_to_gen3_hsc.sh` to create a gen3 repo from the existing gen2 repo (in the same location) and reduce the size of the created sky map.
+* Run `scripts/export_gen3_hsc.py` to export the gen3 repo to `hsc/exports.yaml`.
+
+We only commit `hsc/exports.yaml` to git, because that (and the in-place gen2 files) is all that is necessary to reconstruct a gen3 repo for testing.
```

Comment:
These aren't really gen2 files, they're just files, and with a gen3 export file then it can create a repo that knows where those files are.

Reply:
Try the new wording?